workflow/lib/sbs_process/eval.py

```diff
@@ -0,0 +1,584 @@
+"""Helper functions for evaluating the results of the SBS process steps."""
+
+import string
+
+import pandas as pd
+import numpy as np
+import seaborn as sns
+import matplotlib.pyplot as plt
+import matplotlib.ticker as ticker
+
+
+# helper function to load and concatenate hdfs
+# helper function to load and concatenate HDFs
+def load_and_concatenate_hdfs(hdf_files):
+    """Load and concatenate HDF files from a provided list into a single DataFrame.
```

Comment:
Maybe this is moved. See below comments.

Reply:
Done in https://github.com/cheeseman-lab/brieflow/pull/10/commits/4052c72e7e10a02317e6ad41f35d23ba0a13a276